### PR TITLE
Pod Identifier navigation

### PR DESCRIPTION
**Pull request by brownhoward**
Included how to on navigation to other Pods.
Restructured Trusted App section to reference User Guide.

### CODE REVIEW

File: source/faq.rst [hunk 1]
```diff
@@ -5,23 +5,54 @@ FAQ
...
+
+          * - ``https://podbrowser.inrupt.com``
+
+            - ``Read``, ``Write``, ``Append``, and ``Control``
+
+      #. Click :guilabel:`Add`.
+
+    * If PodBrowser is listed as a trusted application but with the wrong access, select all of the access modes and click :guilabel:`Update`.
+
+.. _`Solid Data Browser`: https://github.com/solid/userguide#data-browser-user-guide
```
Comment (kay-kim) [inline source/faq.rst:43]: Ooh la la -- Mister is getting all fancy with the restructured text 😃  Love it!

File: source/faq.rst [hunk 2]
```diff
@@ -5,23 +5,54 @@ FAQ
...
+
+
+How do I navigate to another Pod?
+=================================
+
+If you need to browse another Pod (or even a Containter in your own Pod), 
+you can navigate directly there by entering the Pod (or Container) URI.
```
Comment (kay-kim) [inline source/faq.rst]: Trivial suggestion

```suggestion
To browse another Pod (or even a Container in your own Pod), you can navigate there directly by entering the Pod (or Container) URI.
```

**Review by kay-kim — APPROVED** [review]
LGTM

File: source/faq.rst (hunk 1 above)
Comment (brownhoward) [reply source/faq.rst:43]: My reptilian brain may be a little slow, but it gets there eventually...

File: source/faq.rst (hunk 2 above)
Comment (brownhoward) [reply source/faq.rst]: Updated

File: source/faq.rst [hunk 3]
```diff
@@ -5,23 +5,54 @@ FAQ
...
+
+       .. list-table::
+
+         * - Application URL
+
+           - Access modes
+
+         * - ``https://podbrowser.inrupt.com``
+
+           - ``Read``, ``Write``, ``Append``, and ``Control``
```
Comment (kay-kim) [inline source/faq.rst]: Oh @kevin-inrupt  - just noticed the .. doesn't line up with A in "At" which is why it's not rendering quite right.

.. should align with the A in At
* - should line up with the l in list-table

```suggestion
        .. list-table::

           * - Application URL

             - Access modes

           * - ``https://podbrowser.inrupt.com``

             - ``Read``, ``Write``, ``Append``, and ``Control``

```

**Pull request merged**